pubsub/mqtt3/metadata.go

```diff
@@ -100,15 +97,6 @@ func parseMQTTMetaData(md pubsub.Metadata, log logger.Logger) (*metadata, error)
 		}
 	}
 
-	m.maxRetriableErrorsPerSec = defaultMaxRetriableErrorsPerSec
-	if val, ok := md.Properties[mqttMaxRetriableErrorsPerSec]; ok && val != "" {
-		var err error
-		m.maxRetriableErrorsPerSec, err = strconv.Atoi(val)
-		if err != nil {
-			return &m, fmt.Errorf("%s invalid maxRetriableErrorsPerSec %s, %s", errorMsgPrefix, val, err)
-		}
-	}
```

Comment:
aren't we removing the property now? I think this is a breaking change?

Reply:
Not necessarily a breaking change.